packages/core/click-handler.js

```diff
@@ -59,7 +59,7 @@ function getResolverUrls(urls) {
       };
     }
 
-    // Extneral urls
+    // External urls
```

Comment:
😆 Today, I installed a spell checker plugin! From now on, you won't see that kind of typos 🤞